dashboard/lib/controllers/run.js

```diff
@@ -0,0 +1,67 @@
+const Run = require('../models/run');
+const db = require('../store');
+const { asyncWrapper } = require('../utils');
+
+const runs = db.getTable('runs');
+
+const api = {
+    insert: asyncWrapper(async (data) => {
```

Comment:
Why do we need `asyncWrapper` here? I would argue it is better to throw the error and let caller handle it rather than catching it. The caller would not know if their call succeeded or failed.